### DISCUSSION TITLE
Error with confidence interval calculations

### DISCUSSION THREAD

**Discussion by theanita1** [msberends/AMR#103] [General]
Hi everyone

Am trying to calculate sir_confidence_intervals and it's throwing the error 
`Error in `summarise()`:
ℹ In argument: `ci_min = sir_confidence_interval(...)`.
ℹ In group 1: `state = ACT`.
Caused by error in `stats::binom.test()`:
! 'n' must be a positive integer >= 'x'`

Whilst I do only have small numbers in my results, I haven't come across this issue with other bug/drug combos with similarly small numbers

Here's my code for what I'm trying to calculate:
sau%>%
  group_by(state)%>%
  summarise(count = n(),
            `no. tested` = n_sir(ERY),
            `no. R` = count_R(ERY),
            ERY = proportion_R(ERY, as_percent = T, minimum = 0), 
            ci_min = sir_confidence_interval(ERY, ab_result = "R", side = "min", as_percent = T, minimum = 1),
            ci_max = sir_confidence_interval(ERY, ab_result = "R", side = "max", as_percent = T, minimum = 1))

and here's my aggregated data 
/state/tested/resistant
- ACT/20/2
- NSW/173/28
- NT/29/6
- Qld/88/14
- SA/57/7
- Tas/15/0
- Vic/152/17
- WA/73/6

**Comment by msberends** [comment]
Hm, I wonder if this isn’t a bug. 

So if I understand well, you have 20 isolates available where `state = "ACT"` of which 2 are resistant? Since susceptibility calculations require >= 30 isolates at default, it could be that the confidence interval calculation fails if this requirement is not met. I’ll look into this.

**Reply by msberends** [reply]
Ah but you have set `minimum = 1`… Weird. I’ll get back to it.

**Reply by theanita1** [reply]
Hi Matthjis

I don't know what changed, but the error has stopped appearing when running the above code, so wanted to give that feedback

Cheers
